src/main.rs

```diff
@@ -10,11 +10,30 @@ use ztunnel::*;
 // #[global_allocator]
 // static GLOBAL: tcmalloc::TCMalloc = tcmalloc::TCMalloc;
 
-#[tokio::main(worker_threads = 2)]
-async fn main() -> anyhow::Result<()> {
+const DEFAULT_WOKRER_THREADS: usize = 2;
+
+fn main() {
     telemetry::setup_logging();
     let config = ztunnel::config::Config {
```

Comment:
I'd like to have all config in Config. can we move it there?

The idea is we will have some more unified way to configure things rather than just env vars in the future, so having everything in one place makes that future proofed.

Reply:
Got the desgin idea, will change it.